src/sealock.h

```diff
@@ -34,6 +35,7 @@ typedef struct sealock_state_struct {
 
 
 int sealock_load_data(sealock_state_t *lock, char *filepath);
```

Comment:
Again, data could mean anything

Reply:
Renamed it to sealock_load_timeseries(). Also renamed similar functions to indicate what they load/set.